src/RecordSession.cc

```diff
@@ -984,7 +992,6 @@ void RecordSession::syscall_state_changed(RecordTask* t,
         return;
       }
       last_task_switchable = PREVENT_SWITCH;
-      t->apply_syscall_entry_regs();
```

Comment:
Uh where did this go?

Reply:
It was incorrect, because this is a ptrace resume, so it already happened on the original entry, and the ptracer may have modified it.